### PR TITLE
Playlist

### PR DESCRIPTION
**Pull request by sanathkumar26700**
1. Added playlist modal in which,
    - User can create playlists
    - Add videos to playlists
    - Remove videos from playlists
2. Added playlist page with playlist cards
    - cards contain delete button to delete the entire playlist
    - Clicking anywhere else takes user to playlist page with video cards of the playlist
3. Added single playlist page,
    - Here users can view their playlist 
    - Delete a particular video from the playlist

### CODE REVIEW

File: src/Components/Playlist modal/playlist-modal.css [hunk 1]
```diff
@@ -0,0 +1,120 @@
+.playlist--section {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
```
Comment (MarkVed17) [inline src/Components/Playlist modal/playlist-modal.css]: I know this works. But you could've even used `inset: 0` shorthand here.

**Review by MarkVed17 — APPROVED** [review]
The favicon scared me at first xD. Nevertheless, it's unique.
 I really liked the Sidebar. Also the playlist implementation.
 
 Some observations:
 The buttons here don't appear aligned.
image

Same on the sign-up page

File: src/Components/Playlist modal/playlist-modal.css (hunk 1 above)
Comment (sanathkumar26700) [reply src/Components/Playlist modal/playlist-modal.css]: Thanks for the suggestion!

**Comment by sanathkumar26700** [comment]
> The favicon scared me at first xD. Nevertheless, it's unique. I really liked the Sidebar. Also the playlist implementation.
> 
> Some observations: The buttons here don't appear aligned. image
> 
> Same on the sign-up page

Appreciate your detailing in pointing out the your observation, will fix it and thanks for the feedback!

**Pull request merged**